<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="MyLabel" name="label_zhangjiakou">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>131</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-3</number>
   </property>
  </widget>
  <widget class="MyLabel" name="label_chengde">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-3</number>
   </property>
  </widget>
  <widget class="MyLabel" name="label_baoding">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-3</number>
   </property>
  </widget>
  <widget class="MyLabel" name="label_langfang">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-3</number>
   </property>
  </widget>
  <widget class="MyLabel" name="label_qinhuangdao">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-3</number>
   </property>
  </widget>
  <widget class="MyLabel" name="label_tangshan">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-3</number>
   </property>
  </widget>
  <widget class="MyLabel" name="label_xingtai">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-3</number>
   </property>
  </widget>
  <widget class="MyLabel" name="label_hengshui">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-3</number>
   </property>
  </widget>
  <widget class="MyLabel" name="label_cangzhou">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-3</number>
   </property>
  </widget>
  <widget class="MyLabel" name="label_shijiazhuang">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-3</number>
   </property>
  </widget>
  <widget class="MyLabel" name="label_handan">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>440</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-3</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
   <slots>
    <signal>Clicked(MyLabel*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
